### PR TITLE
feat: Allow --force argument on deletion

### PR DESCRIPTION
**Pull request by mfornet**
Skip confirmation if --force is specified during deletion.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -233,8 +233,8 @@ exports.deleteAccount = async function (options) {
             throw e;
         }
     }
-
```
Comment (volovyks) [inline index.js:236]: Let's avoid formatting changes.

File: index.js [hunk 2]
```diff
@@ -233,8 +233,8 @@ exports.deleteAccount = async function (options) {
             throw e;
         }
     }
-    
-    if (await confirmDelete()) {
+
+    if (options.force || await confirmDelete()) {
```
Comment (volovyks) [inline index.js:237]: It would be cool to add a note about this flag in README.
Small test is also appritiated.

Comment (mfornet) [reply index.js:237]: Added note on README. I tried to run integration tests locally, but doing `yarn test` is insufficient. I guess I need to spin off a NEAR node locally to make it work.

File: index.js (hunk 1 above)
Comment (mfornet) [reply index.js:236]: Reverted

File: index.js (hunk 2 above)
Comment (volovyks) [reply index.js:237]: It should work with testnet, but ok.

**Review by volovyks — APPROVED** [review]
Thnaks @mfornet !

**Pull request merged**